src/slidy/slidy.js

```diff
@@ -100,6 +100,17 @@ export function slidy (slider, options) {
     slideContainerDOMEl.style.cssText = cssText
   }
 
+  function _setTailArrowClasses () {
+    const { tailArrowClass, infinite } = options
+    if (tailArrowClass === undefined || tailArrowClass === '' || infinite) { return }
+
+    const prevArrow = frameDOMEl.getElementsByClassName(options.classNamePrevCtrl)[0]
+    prevArrow.classList[ index < 1 ? 'add' : 'remove' ](tailArrowClass)
+
+    const nextArrow = frameDOMEl.getElementsByClassName(options.classNameNextCtrl)[0]
+    nextArrow.classList[ index > options.items.length - 2 ? 'add' : 'remove' ](tailArrowClass)
```

Comment:
options.items.length - 1 is the last item in the array, so, when our position is *higher* than -2 then we must add the class.
Maybe it would have been clearer with index *>=* .... ?